crates/sui-rosetta/src/types.rs

```diff
@@ -663,7 +663,8 @@ pub struct ConstructionMetadata {
     pub sender: SuiAddress,
     pub coins: Vec<ObjectRef>,
     pub objects: Vec<ObjectRef>,
-    pub total_coin_value: u64,
+    #[serde(with = "str_format")]
```

Comment:
Minor: Wouldn't here `#[serde(with = "str_format")]` suffice? Do we also need to change to i128?